framework/aster-frame/src/vm/config.rs

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! A configuration module for framework memory management.
+
+use super::Vaddr;
+
+pub const KERNEL_STACK_SIZE: usize = PAGE_SIZE * 64;
+pub const KERNEL_HEAP_SIZE: usize = PAGE_SIZE * 256;
```

Comment:
The name `KERNEL_HEAP_SIZE` is not quite accurate as the heap size can grow. So `KERNEL_INIT_HEAP_SIZE` or `KERNEL_MIN_HEAP_SIZE` makes more sense.